.github/workflows/misspell.yml

```diff
@@ -8,13 +8,26 @@ concurrency:
 
 jobs:
   static-checks:
-    runs-on: ubuntu-latest
+    runs-on: edge-example-ci
```

Comment:
What is edge-example-ci ? A selfhosted runner?

Reply:
Yes, there's two of them running in Mercury now.